dts/arm/nxp/nxp_s32z27x_r52.dtsi

```diff
@@ -0,0 +1,407 @@
+/*
+ * Copyright 2022 NXP
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <mem.h>
+#include <arm/armv8-r.dtsi>
+#include <zephyr/dt-bindings/interrupt-controller/arm-gic.h>
+
+/ {
+	cpus {
+		#address-cells = <1>;
+		#size-cells = <0>;
+
+		cpu@0 {
+			device_type = "cpu";
+			compatible = "arm,cortex-r52";
+			reg = <0>;
+		};
+
+		cpu@1 {
```

Comment:
Just out of curiosity, do you have booted all 8 cores but there is no SMP supported for v8R32?

Reply:
Not all the cores are boot, only the core on which Zephyr is running single-core. There's the possibility to chose the target core providing different arguments to the west runner as described in https://github.com/zephyrproject-rtos/zephyr/pull/49161/commits/9a7af8f9818f0101b0cbc496017bcf266adccebf#diff-b226f398a3c6bd057366b1fa27e22cac9beac761d9a16e4cab0e36bd9a42e2b7R191-R201

Reply:
Thanks for infomation